pkg/auth/auth_test.go

```diff
@@ -0,0 +1,66 @@
+package auth
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/containers/image/v5/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAuthConfigsToAuthFile(t *testing.T) {
+	for _, tc := range []struct {
+		name             string
+		server           string
+		shouldErr        bool
+		expectedContains string
+	}{
+		{
+			name:             "empty auth configs",
+			server:           "",
+			shouldErr:        false,
+			expectedContains: "{}",
+		},
+		{
+			name:             "registry with prefix",
+			server:           "my-registry.local/username",
+			shouldErr:        false,
+			expectedContains: `"my-registry.local/username":`,
+		},
+		{
+			name:             "normalize https:// prefix",
+			server:           "http://my-registry.local/username",
+			shouldErr:        false,
+			expectedContains: `"my-registry.local/username":`,
```

Comment:
This is inconsistent with https://github.com/containers/image/blob/4fcb52359b759a276a760fb28c939c357b3aea48/pkg/docker/config/config_test.go#L938 .